data/constants/keycodes/keycodes_0.0.1_user.hjson

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
The semantics here are really muddy. `QK_USER_0` is now being used to implement keyboard level custom keycodes. As for the solution, I will have to cycle back to it when i have a bit more free time.

Reply:
> The semantics here are really muddy. `QK_USER_0` is now being used to implement keyboard level custom keycodes. As for the solution, I will have to cycle back to it when i have a bit more free time.

I mean, we have separate kb and user blocks, now. Using the existing KB block would allow users/manus/vendors to specify the behavior for both via and non-via functionality without the need for preprocessor hacks that are present in many boards.  (it would also reduce the number of used keycodes)

Reply:
KB for keyboard, USER for keymap was always the long term intention. However getting there is a little muddy due to the mess that existing in the repo. There were a few scenarios where it would crap out due to existing assumptions of keycode order.

Reply:
We could change it to using the `QK_KB` `QK_KB_MAX` range instead?

Reply:
OK, changed it to `QK_KB_0`...